src/GitWrapper/GitRepository.php

```diff
@@ -3,6 +3,8 @@
 
 class GitRepository
 {
+    const TAG_ORIGIN = 'tags';
```

Comment:
As this doesn't belong to the function, I would kindly ask you to remove the unnecessary constant.